### DISCUSSION TITLE
Google Cloud Install - Let’s Encrypt Expiry

### DISCUSSION THREAD

**Discussion by JonLee15** [NightscoutFoundation/xDrip#3778] [Q&A]
Hi all,

I have Nightscout set up on google cloud. I received an email that my Let’s Encrypt certificate will expire in 6 days. Any advice on how to renew?

I did discover today that my FreeDNS account went dormant, unsure if that’s relevant or not. I reactivated it. 

Thanks! 
![IMG_0395](https://github.com/user-attachments/assets/a6cda13d-b673-4b5f-8b9f-95a02e2d434a)

**Comment by Navid200** [comment]
If your Nightscout is running, you can ignore the warning about the certificate expiry.

With respect to FreeDNS, you need to log into your account twice a year.  And an email is sent to you after 5.5 months to remind you.  You can make sure that it won't go to your junk mail folder next time.
